org.eclipse.january.examples/src/org/eclipse/january/examples/dataset/AxesMetadataExample.java

```diff
@@ -278,10 +282,10 @@ else if (bMin < aMin && bMax > aMax)
 							}
 
```

Comment:
Change from using indexes to positions `int aStart` to `int[] aStart` and then use:
```
aStart = aIndices.getNDPosition(DatasetUtils.findIndexGreaterThanOrEqualTo(aIndices, startPoint));

aEnd = aReverse.getNDPosition(DatasetUtils.findIndexLessThanOrEqualTo(aReverse, endPoint));

aEnd[0]++; // adjust end point
```